auction-server/src/opportunity/service/add_opportunity.rs

```
@@ -20,7 +20,7 @@ pub struct AddOpportunityInput {
 #[derive(Debug, Clone)]
 enum OpportunityAction {
     Add,
-    Refresh(OpportunitySvm),
+    Refresh(Box<OpportunitySvm>),
```

Why boxed?

---

clippy and enum size, its an easy change and keeps this enums size to like n bytes instead of 400

---

nit: While this will put things inside heap, maybe it's a better idea to ask clippy to ignore the error for this line